auth/src/it/scala/janstenpickle/vault/auth/UserPassIT.scala

```diff
@@ -30,37 +30,37 @@ class UserPassIT extends VaultSpec with ScalaCheck {
   def removeClient(client: String) =
     authAdmin.disable(client).attemptRun(_.getMessage()) must beOk
 
-  def authPass = test((username, password, client, ttl) =>
+  def authPass = test((username, password, client, ttl) ⇒
```

Comment:
I don't really like the use of `⇒` over `=>`. I realise this comes down to a matter of preference, but a style change like this should really be discussed before including in a pull request with feature code :)